addons/account/models/account_move.py

```diff
@@ -786,6 +795,16 @@ def _compute_is_being_sent(self):
         for move in self:
             move.is_being_sent = bool(move.sending_data)
 
+    @api.depends('is_being_sent', 'is_move_sent')
```

Comment:
This is enough :smile: We don't care about the move being sent asynchronously :smile: 
```suggestion
    @api.depends('is_move_sent')
```